src/aca_main.cpp

```diff
@@ -217,6 +217,11 @@ int main(int argc, char *argv[])
   g_grpc_server_thread =
           new std::thread(std::bind(&GoalStateProvisionerImpl::RunServer, g_grpc_server));
   g_grpc_server_thread->detach();
+  rc = aca_ovs_l2_programmer::ACA_OVS_L2_Programmer::get_instance().setup_ovs_bridges_if_need();
+  if (rc == EXIT_FAILURE) {
+      ACA_LOG_ERROR("%s \n", "ACA is not able to create the bridges, please check your environment");
```

Comment:
Since the above lines have created some resources such as `g_grpc_server` and `g_grpc_server_thread`, maybe it will be nice to call `aca_cleanup()` here, before returning the `rc`.

Reply:
Can we get more traction on this PR?